releases/note-source/v16_0.rst

```diff
@@ -29,6 +29,21 @@ Summer 2018 Release (v16_0)
 These release notes highlight significant changes to the Science Pipelines codebase which are likely to be of wide interest.
 For a complete list of changes, see :ref:`release-v16-0-tickets`.
 
+.. _release-v16-0-scope:
+
+Release 16.0 Scope
+------------------
+
+The scope of the release is to be executed on a controlled environment.
+It has been proven to run on newinstall.sh environment, and it is expected
```

Comment:
I'm not entirely sure what's being said here. This is fairly legalistic language that's obfuscating the actual issue and preventing the user from making an appropriate decision.

I think what you're trying to say is "This release is only known to work if you install it according to the <Install with newinstall.sh and eups distrib> instructions." And add the demo package failure as a known issue.